py/torch_tensorrt/dynamo/conversion/plugins/_generate_plugin.py

```diff
@@ -0,0 +1,176 @@
+from sympy import lambdify
+
+def mksym(shape_env, value, source, dynamic_dim):
+    return shape_env.create_symintnode(
+        shape_env.create_symbol(
+            value,
+            source=source,
+            dynamic_dim=dynamic_dim,
+        ),
+        hint=value,
+        source=source,
+    )
+
+def generate_plugin(plugin_name : str):
+    namespace, name = plugin_name.split("::")
+    
+    # retrieve the corresponding torch operation using the passed in string
+    torch_op = getattr(getattr(torch.ops, namespace), name)
+    
+    # helper function that generates the required signature based on the torch operation
+    def generate_signature(torch_op):
+        schema = torch_op._schemas['']
+        tensor_args = []
+        arg_list = []
+        
+        args = []
+        kwargs = []
+        
+        register_func_annotation = {}
+        impl_func_annotation = {}
+        
+        for arg in schema.arguments:
+            arg_list.append(arg.name)
+            
+            if arg.type.isSubtypeOf(torch._C.TensorType.get()):
+                tensor_args.append(arg)
+                register_func_annotation[arg.name] = trtp.TensorDesc
+                impl_func_annotation[arg.name] = trtp.Tensor
+            elif arg.type.isSubtypeOf(torch._C.FloatType.get()):
+                register_func_annotation[arg.name] = float
+                impl_func_annotation[arg.name] = float
+            elif arg.type.isSubtypeOf(torch._C.IntType.get()):
+                register_func_annotation[arg.name] = int
+                impl_func_annotation[arg.name] = int
+            else:
+                raise ValueError("arg type is not handled")
+                
+                
+            if arg.default_value is None:
+                args.append(arg.name)
+            else:
+                kwargs.append(f"{arg.name} = {arg.default_value}")
+         
+        input_signature = ", ".join(arg_list)
+        plugin_signature = f"def add_plugin_desc({input_signature}):"
+        args_input = ", ".join(args)
+        kwargs_input = ", ".join(kwargs)
+
+        plugin_impl_arg_list = arg_list
+        plugin_impl_arg_list.append('outputs')
+        plugin_impl_arg_list.append('stream')
+        plugin_impl_input = ", ".join(plugin_impl_arg_list)
+        plugin_impl_signagture = f"def add_plugin_impl({plugin_impl_input}):"
+        
+        
+        register_func_annotation["return"] = Tuple[trtp.TensorDesc]
+        
+        impl_func_annotation["outputs"] = Tuple[trtp.Tensor]
+        impl_func_annotation["stream"] = int
+        
+        return args_input, kwargs_input, plugin_signature, plugin_impl_signagture, register_func_annotation, impl_func_annotation
+        
+    # Use the helper function to get the required signatures
+    args_input, kwargs_input, plugin_signature, plugin_impl_signature, register_func_annotation, impl_func_annotation = generate_signature(torch_op)
+    print(args_input)
```

Comment:
Make this debug info

Reply:
@narendasan  when I use: 
```
    _LOGGER.debug(f"Plugin registration function: \n{codegen_plugin}")
```
It won't print anything. How to resolve this?